shell/common/persistent_cache.h

```diff
@@ -73,6 +73,7 @@ class PersistentCache : public GrContextOptions::PersistentCache {
   static void MarkStrategySet() { strategy_set_ = true; }
 
   static constexpr char kSkSLSubdirName[] = "sksl";
+  static constexpr char kAssetFileName[] = "shaders.json";
```

Comment:
This seems like an all too common name and may collide with other assets. How about `io.flutter.shaders.json` or something more indicative of its purpose?

Reply:
Done.